jdaviz/configs/mosviz/helper.py

```diff
@@ -419,6 +419,14 @@ def load_data(self, spectra_1d=None, spectra_2d=None, images=None,
             String representing the label for the data item loaded via
             ``images``. Can be a list of strings representing data labels
             for each item in ``images`` if  ``images`` is a list.
+
+        directory: str, optional
```

Comment:
A little confusing here since it is not in the signature above, but rather extracted from `kwargs` but I guess we can clean up the docstring in a future PR if needed.
```suggestion
        directory : str, optional
```